src/component/variantPage/biologicalFunction/Oncokb.tsx

```diff
@@ -14,7 +14,7 @@ interface IOncokbProps {
 export enum ONCOGENICITY {
     ONCOGENIC = 'Oncogenic',
     LIKELY_ONCOGENIC = 'Likely Oncogenic',
-    PREDICTED_ONCOGENIC = 'Predicted Oncogenic',
+    RESISTANCE = 'Resistance',
```

Comment:
Ideally we should be getting these type of constants from an external OncoKB package or something like that.

Reply:
@onursumer oh absolutely. We are going to move OncoKB Card as separate component and transfer the ownership to OncoKB as well. Then we will set up the constants there. oncokb-commons? :)